.github/workflows/run_ct_test.yaml

```diff
@@ -2,6 +2,15 @@ name: Run CT Tests
 on: [push, pull_request, workflow_dispatch]
 jobs:
   run_ct_test:
-    uses: Cray-HPE/hms-build-image-workflows/.github/workflows/run_ct_test.yaml@v1
+    uses: Cray-HPE/hms-build-image-workflows/.github/workflows/run_ct_test_with_hms_sim.yaml@CASMHMS-5796
```

Comment:
Once https://github.com/Cray-HPE/hms-build-image-workflows/pull/8 merges need to change the tag ref to `v1`